sw/device/silicon_creator/manuf/base/perso_blob.h

```diff
@@ -0,0 +1,30 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_MANUF_BASE_PERSO_BLOB_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_MANUF_BASE_PERSO_BLOB_H_
+
+#include <stdbool.h>
+#include <stddef.h>
+
+#include "sw/device/lib/base/status.h"
+
+/**
+ * Wrap the passed in certificate in a perso LTV object and copy it into the
+ * body of the perso_blob.
+ *
+ * @param name the name of the certificate
+ * @param needs_endorsement defines the type of the LTV object the certificate
+ *              is wrapped into
+ * @param cert_body the actual certificate
+ * @param cert_size size of the certificate in bytes
+ *
+ * @return status of the operation.
+ */
+extern status_t prepare_cert_for_shipping(const char *name,
```

Comment:
I think it would be better if we just had one `perso_ltv_data.h` (or named something like that) lib that combines `personalization_data.h` with this `perso_blob.h`. This lib could contain all the formatting / parsing functions.

This function could take a perso_blob_t as an outparam.

Reply:
done